content/en/developers/smart-contracts/how-tos/get-test-funds/index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Get test funds"
+description: "Test funds are available to developers so they can test their smart contracts and applications within the confines of a test network."
+lead: "Test funds are available to developer so that they can test their smart contracts and applications within the confines of a test network. The process for getting test funds differs across test networks. This page covers how to get test funds from the Hyperspace testnet as well as from a local testnet."
+menu:
+    developers:
+        parent: "developers-how-tos"
+        identifier: "get-test-funds-sfh2o9fbaiubo4igeirbuyat"
+type: docs
+weight: 40
+draft: false
+images: []
+aliases:
+    - "/developers/smart-contracts/how-tos/use-a-faucet/"
+    - "/fvm/how-tos/use-a-faucet/"
+---
+
+## Hyperspace testnet
+
+Before we begin, you must have an address on the Hyperspace testnet. Follow the [Add to MetaMask guide]({{< relref "add-to-metamask" >}}) if you don't have one yet.
```

Comment:
Devs can have t4 addresses/ETH addresses, and they can have t1 & t3 addresses as well. Creating a wallet in metamask is one option. Should we also mention that devs can also request testnet tokens using t1&t3 addresses?

Reply:
Yeah that's probably good to call out. We don't have to go into a massive amount of detail here, but we can mention that alternative account types, and link to the accounts page.

Reply:
Resolved in a4988093.